packages/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -161,7 +161,8 @@ export class TriggerEvent {
             to: subscriber,
             transactionId: command.transactionId,
             userId,
-            ...(actor && actorProcessed && { actor: actorProcessed }),
+            ...(actor &&
+              actorProcessed && { actor, actorSubscriber: actorProcessed }),
```

Comment:
Here, revert back please to:
`...(actor && actorProcessed && { actor: actorProcessed }),`

The actorProcessed already has the subscriberId (which is the only thing in the actor object)